R/plumber.R

```diff
@@ -887,9 +887,20 @@ Plumber <- R6Class(
     #' @param api This can be
     #'   * an OpenAPI Specification formatted list object
     #'   * a function that accepts the OpenAPI Specification autogenerated by `plumber` and returns a OpenAPI Specification formatted list object.
+    #'   * a path to an OpenAPI Specification
     #'
     #'  The value returned will not be validated for OAS compatibility.
     setApiSpec = function(api = NULL) {
+      if (is.character(api)) {
```

Comment:
```suggestion
      if (is.character(api) && length(api) == 1 && file.exists(api)) {
```

Reply:
I'll have to check, I was not sure about checking for file.exist since it returns FALSE is you use a URL (non local file)